commit ee8a64190b62a083e1c10c4949ca1e296befbfd3
Author: Felix Yan <felixonmars@archlinux.org>
Date:   Wed May 2 14:54:06 2018 +0800

    fix: build failure due to missing xi in pkgconfig array
    
    keyboardmonitor made use of several XI functions without having xi added
    to the pkgbuild array. This causes a build failure about undefined
    references to those functions. Adding xi fixes it here.
    
    Change-Id: Ifb8171c085c30b014e396376e743ff0ef1fea3b0

diff --git a/src/widgets/private/keyboardmonitor/keyboardmonitor.pri b/src/widgets/private/keyboardmonitor/keyboardmonitor.pri
index 3827ce4..06b0862 100644
--- a/src/widgets/private/keyboardmonitor/keyboardmonitor.pri
+++ b/src/widgets/private/keyboardmonitor/keyboardmonitor.pri
@@ -1,5 +1,5 @@
 CONFIG += c++11 link_pkgconfig
-PKGCONFIG += xext x11
+PKGCONFIG += xext x11 xi
 
 INCLUDEPATH += $$PWD
 
